packages/@icon-magic/config-reader/src/config-validator.ts

```diff
@@ -0,0 +1,32 @@
+import { configSchema } from './schemas/config-schema';
+import * as Ajv from 'ajv';
+
+let metaSchema = require('ajv/lib/refs/json-schema-draft-04.json');
+
+/**
+ * Validates the top level properties of the config object.
+ * @param config The config object to validate.
+ */
+export function validateConfigSchema(config: object): Boolean {
+  return validateSchema(config, configSchema);
+}
+
+export function validateSchema(data: object, schema: object) {
```

Comment:
Should this be non-exported? Would you ever want to use `validateSchema` directly instead of `validateConfigSchema`?

Reply:
Yes, I want to be able to do that before editing the config file or creating a new one as a sanity check.

Reply:
Just make sure you throw in a tsdoc comment :)